web/templates/base.html

```diff
@@ -14,137 +14,31 @@
         <!-- Bootstrap CSS -->
         <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='bootstrap-5.1.3-dist/css/bootstrap.min.css') }}">
 
+        <!-- Fonts -->
         <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='fontawesome-free-6.0.0-web/css/all.min.css') }}">
         <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/opensans.min.css') }}">
 
+        <!-- CSS -->
         <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/main.css') }}">
+        <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/plotly.css') }}">
+
+        <!-- Bootstrap Optional JavaScript -->
+        <script type="text/javascript" src="{{ url_for('static', filename='bootstrap-5.1.3-dist/js/bootstrap.bundle.min.js') }}"></script>
+
+        <!-- JQuery Javascript -->
+        <script type="text/javascript" src="{{ url_for('static', filename='js/jquery-3.6.0.min.js') }}"></script>
+
+        <!-- Plotly JavaScript -->
+        <script type="text/javascript" src="{{ url_for('static', filename='js/plotly-2.11.1.min.js') }}"></script>
```

Comment:
Just a note here, I don't have plotly.js included in my base template because it's huge. Only pages that need it include it. That's probably not a big deal given the efficient caching performance of modern browsers, but it's something to consider.